src/NewRelic.Telemetry/Logging/TelemetryLogging.cs

```diff
@@ -27,32 +27,32 @@ private static string MessageFormatter(object state, Exception error)
             return $"{Prefix} {state} {error}".Trim();
         }
 
-        internal TelemetryLogging(ILoggerFactory? loggerFactory)
+        public TelemetryLogging(ILoggerFactory? loggerFactory)
```

Comment:
Makes me wonder if the `TelemetryLogging` class should always be internal. For direct consumers of the telemetry SDK, is there a need to have access to this class?

Reply:
You're correct this type doesn't need to be public.